test/source/tests/setup.ts

```diff
@@ -415,6 +417,25 @@ AN8G3r5Htj8olot+jm9mIa5XLXWzMNUZgg==
       await settingsPage.close();
     }));
 
+    ava.default('user@forbid-storing-passphrase-org-rule.flowcrypt.test - do not store passphrase', testWithBrowser(undefined, async (t, browser) => {
+      const acctEmail = 'user@forbid-storing-passphrase-org-rule.flowcrypt.test';
+      const settingsPage = await BrowserRecipe.openSettingsLoginApprove(t, browser, acctEmail);
+      await Util.sleep(5);
+      const passphrase = 'long enough to suit requirements';
+      await SetupPageRecipe.createKey(settingsPage, 'unused', 'none', { key: { passphrase }, usedPgpBefore: false });
```

Comment:
On the setup page, we should also add a way to test that the appropriate checkbox is disabled. (and add similar tests for other setup scenarios, like importing key)

Reply:
Done